Automattic/simplenote-android
# Update Icon Background

### Fix
Update the launcher icon to force a light background.  We've received reports of the launcher icon background being dark and unable to be light regardless of the Android dark mode setting.  We haven't been able to reproduce the issue, but we believe it's more common on Samsung devices with One UI.

### Test
1. Disable dark mode.
2. Open app drawer.
3. Notice Simplenote icon is white.
3. Enable dark mode.
4. Open app drawer.
5. Notice Simplenote icon is white.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
